lib/mongoid/association/embedded/batchable.rb

```diff
@@ -85,7 +85,8 @@ def batch_remove(docs, method = :delete)
         def batch_replace(docs)
           if docs.blank?
             if _assigning? && !empty?
-              _base.delayed_atomic_sets.clear
+              _base.delayed_atomic_sets.delete(path)
```

Comment:
> I think this should be backported to 7.3 & 7.4 if the commit backports.

@p-mongo I see this change was backported to 7.3 & 7.4, but it is **not** on the `7.5-stable` branch

Can this be updated on 7.5, also?

Reply:
@comandeo just making sure you see this!

Reply:
https://github.com/mongodb/mongoid/pull/5646 cherry-picks the change to `7.5-stable`